ci/travis-install.sh

```diff
@@ -1,8 +1,5 @@
 #!/bin/bash
 set -ev
 
-if [ "$TRAVIS_NODE_VERSION" == "v0.12.17" ]; then 
-    npm install --force && npm link && npm link webpack;
-else 
-    npm install yarn -g && yarn install && yarn link || true && yarn link webpack;
-fi
+npm install yarn -g && yarn install && yarn link || true && yarn link webpack;
```

Comment:
> If your Node.js project has the yarn.lock file in the repository root, we proceed to install Yarn and execute the yarn command instead of the default npm install.

We'd want to revisit (down the road) if we want to actually include lock file again. There was some discussion on this.